### PR TITLE
footer component

### LINKED ISSUE #16: make footer
make a separate "footer" component
- put some generic footer stuff (social media icons)
- here is a list of links to assign to icons
'https://www.facebook.com/cutiehack/'
'https://www.instagram.com/cutiehack_ucr/'
'https://www.linkedin.com/company/17907222/admin/'

### PR DESCRIPTION
**Pull request by vwong031**
image

### CODE REVIEW

**Comment by minsooerickim** [comment]
curious, did ur own prettier config in vscode extension or something, cuz it looks like whatever prettier thing u used changed files u shouldn't have; Rn, we only run prettier against `components/*` and` pages/*` cuz some files get messed up with prettier.

can u undo ur prettier changes and re run it with our script, `yarn format` (its defined under `package.json` if u wanna take a look)

File: components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import Link from 'next/link'
+import { FaFacebook, FaInstagram, FaLinkedin, FaTwitter } from 'react-icons/fa'
+
+import { MdEmail } from 'react-icons/md'
+
+const SocialLinks = [
+  {
+    path: 'https://www.facebook.com/cutiehack/',
+    title: 'Facebook',
+    image: <FaFacebook size="56"></FaFacebook>,
```
Comment (minsooerickim) [inline components/Footer.tsx:11]: i dont think we need the size tbh. it looks pretty big rn and i think we can just use whatever the default size (without specifying size)

**Comment by minsooerickim** [comment]
everything else looks good tho, noice work

**Comment by vwong031** [comment]
> curious, did ur own prettier config in vscode extension or something, cuz it looks like whatever prettier thing u used changed files u shouldn't have; Rn, we only run prettier against `components/*` and` pages/*` cuz some files get messed up with prettier.
> 
> can u undo ur prettier changes and re run it with our script, `yarn format` (its defined under `package.json` if u wanna take a look)

oo, ya i got confused on how to run prettier with yarn so i did it with npm like i normally do. i didn't realize it would break some stuff

File: components/Footer.tsx (hunk 1 above)
Comment (vwong031) [reply components/Footer.tsx:11]: i removed the size, but imma add it back cuz it looks like this without it
image

Comment (minsooerickim) [reply components/Footer.tsx:11]: ah oki oki

File: components/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    title: 'LinkedIn',
+    image: <FaLinkedin size="56"></FaLinkedin>,
+  },
+  {
+    path: 'https://twitter.com/citrushack',
+    title: 'Twitter',
+    image: <FaTwitter size="56"></FaTwitter>,
+  },
+  {
+    path: 'citrushack@gmail.com',
```
Comment (minsooerickim) [inline components/Footer.tsx]:
```suggestion
    path: 'mailto:citrushack@gmail.com',
```

**Pull request merged**